audio/beets/Portfile

```diff
@@ -0,0 +1,518 @@
+                    It catalogs your collection, automatically improving its metadata as it goes. \
+                    It then provides a bouquet of tools for manipulating and accessing your music.
+
+homepage            http://beets.io/
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+checksums           rmd160  b78e41d04dc9a9199696e5c31d885e3503b54d2a \
+                    sha256  887f7bbac0fc14c49469e50d406fd216f914a27acf3818c6503c223f9825342b \
+                    size    1694845
+
+python.default_version 39
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+if {${name} eq ${subport}} {
+    depends_lib-append \
+                    port:py${python.version}-beautifulsoup4 \
+                    port:py${python.version}-Pillow \
+                    port:py${python.version}-musicbrainzngs \
+                    port:py${python.version}-jellyfish \
+                    port:py${python.version}-six \
+                    port:py${python.version}-mediafile \
+                    port:py${python.version}-confuse \
+                    port:py${python.version}-munkres \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-requests-oauthlib \
+                    port:py${python.version}-unidecode \
+                    port:py${python.version}-acoustid \
+                    port:py${python.version}-yaml \
+                    port:py${python.version}-gmusicapi \
+                    port:py${python.version}-discogs-client \
+                    port:py${python.version}-last \
+                    port:py${python.version}-langdetect \
+                    port:py${python.version}-mpd2 \
+                    port:py${python.version}-flask \
+                    port:py${python.version}-flask-cors \
+                    port:py${python.version}-rarfile \
+                    port:py${python.version}-xdg \
+                    port:py${python.version}-soco \
+                    port:py${python.version}-mutagen \
+                    port:py${python.version}-reflink
+
+    depends_run-append \
+                    port:chromaprint \
+                    port:ffmpeg \
+                    port:flac \
+                    port:mp3val \
+                    port:ImageMagick \
+                    port:ipfs
+
+    post-destroot {
+        xinstall -d ${destroot}${prefix}/share/man/man1
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.1] ${destroot}${prefix}/share/man/man1/
+
+        xinstall -d ${destroot}${prefix}/share/man/man5
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.5] ${destroot}${prefix}/share/man/man5/
+
+        xinstall -d ${destroot}${prefix}/share/${name}
+        xinstall -m 0644 ${worksrcpath}/README.rst ${destroot}${prefix}/share/${name}
+        xinstall -m 0644 ${worksrcpath}/LICENSE ${destroot}${prefix}/share/${name}
```

Comment:
you can use `xinstall -m 0644 -W ${worksrcpath} file1 file2 ${destroot}${docdir}`